app/services/recruitment_cycle_creation_service.rb

```diff
@@ -10,12 +10,12 @@ def initialize(year:, application_start_date:, application_end_date:)
   end
 
   def call
-    RecruitmentCycle.create!(
+    recruitment_cycle = RecruitmentCycle.create!(
       year: @year,
       application_start_date: @application_start_date,
       application_end_date: @application_end_date,
     )
 
-    RolloverJob.perform_later
+    RolloverJob.perform_later(recruitment_cycle.id)
```

Comment:
I wonder if we should add defensive measures here? If a user creates a `RecruitmentCycle` with the year being 2-3 years in the future, should the rollover script be run?

Reply:
> I wonder if we should add defensive measures here? If a user creates a `RecruitmentCycle` with the year being 2-3 years in the future, should the rollover script be run?

If we wanna go this way, I'd recommend to not run the rollover at all when a cycle is created and we should add a button "Run rollover" with confirmation - on a recruitment cycle index/show action.

Reply:
👍 I think I'm good with that too. We can be explicit.

Reply:
I will create a card then to add the button with confirmation and merge this PR.  Thanks!